react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx

```diff
@@ -695,3 +698,99 @@ MultiColorMap.args = {
         ],
     },
 };
+
+// colorselector for welllayer
+const wellLayers = [
+    {
+        "@@type": "WellsLayer",
+        data: "@@#resources.wellsData",
+        logColor: [exampleData[0].layers[4]][0].logColor,
```

Comment:
Please avoid using the exampleData json object. It is not a common use case to use json to specify the props, so it distracts from the purpose of the story. Please take a look at the Volve wells example (https://equinor.github.io/webviz-subsurface-components/storybook-static/?path=/story/deckglmap-wells-layer--volve-wells), where the props are defined as js objects. It makes the example easier to understand.